samples/unit-tests/boost/tooltip/demo.js

```diff
@@ -40,3 +40,76 @@ QUnit.test('Tooltip on a boosted chart with categories', function (assert) {
         'The last hoverable point should be the last in the series. (#18856)'
     );
 });
+
+QUnit.test(
+    'Scatter with boost should show correct values in tooltip (#20621)',
+    function (assert) {
+
+        Highcharts.addEvent(Highcharts.Series, 'setOptions', e => {
+            const series = e.target;
+            const yAxis = series.chart.axes[1];
+            yAxis.treeGrid = false;
+        });
+
+        var chart = Highcharts.chart('container', {
```

Comment:
Probably does not matter much if at all but `const chart`" is more appropriate since `chart` is not reassigned within the scope and this allows the engine to optimize around that. 

(I know there is tons of `var chart` in the tests but iirc we are moving away from it)